ledger/test-common/BUILD.bazel

```diff
@@ -99,8 +100,8 @@ general_test_names = [
 
 # Correspond to the directories under src/test/lib/daml
 test_names = {
-    version: general_test_names + (carbon_test_names if alias == "dev" else [])
-    for alias, version in lf_version_configuration.items()
+    lf_version: general_test_names + (carbon_test_names if versions.gte(lf_version, "1.15") else [])
+    for lf_version in lf_version_configuration_versions
```

Comment:
fyi, you cannot use `lf_version_configuration.items` here, because values can be repeated.